flipperdevices/flipperzero-firmware
# Calibrate HSE xtal, adjust on-chip load capacitors

# What's new

- HSE xtal capacitor configuration

# Verification

- Compile and upload
- Remove C9/C11 according to errata 1: https://flipperzero.atlassian.net/wiki/spaces/FLIP/pages/1120862217/7.F5B6C4+Errata
- Check xtal frequency, should be 32mHz +/-20ppm

# Checklist (do not modify)

- [x] Cross-check
